src/index.tsx

```diff
@@ -87,6 +114,26 @@ export async function trackEvent(
   return await PiwikProSdk.trackEvent(category, action, { name, value });
 }
 
+/**
+ * Set a custom dimension.
```

Comment:
Might be good to add some mention about how the function itself does not send anything to the analytics server, and also something about how the dimension is applied to subsequent tracking calls.

Also good to mention the role of the scope argument on different platforms.